jkube-kit/jkube-kit-spring-boot/src/main/java/org/eclipse/jkube/springboot/enricher/SpringBootHealthCheckEnricher.java

```diff
@@ -112,8 +114,9 @@ protected Probe buildProbe(Integer initialDelay, Integer period, Integer timeout
               springBootConfiguration.getManagementContextPath() : "";
         } else {
             scheme = StringUtils.isNotBlank(springBootConfiguration.getServerKeystore()) ? SCHEME_HTTPS : SCHEME_HTTP;
-            prefix = StringUtils.isNotBlank(springBootConfiguration.getServerContextPath()) ?
-              springBootConfiguration.getServerContextPath() : "";
+            String basePath = hasSpringWebFluxDependency(getContext().getProject()) ? springBootConfiguration.getWebFluxBasePath() :
+              springBootConfiguration.getServerContextPath();
+            prefix = StringUtils.isNotBlank(basePath) ? basePath : "";
             prefix += StringUtils.isNotBlank(springBootConfiguration.getServletPath()) ?
               springBootConfiguration.getServletPath() : "";
             prefix += StringUtils.isNotBlank(springBootConfiguration.getManagementContextPath()) ?
```

Comment:
for clarity it might be better not to use so many ternary operators.

Something that shows clear precedence as: if hasWebFlux and webfluxpath is not blank, else if servlet path is not blank, and so on... will make things much easier to understand